stable/wordpress/templates/tls-secrets.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.ingress.enabled }}
+{{- range .Values.ingress.secrets }}
```

Comment:
This is what confused me (r.e. secrets being in a host object), looks like this template is wrong.

Reply:
No, the idea here is that secrets is a separate array from the hosts.  There is a possibility of two hosts using the same certificate (a wildcard certificate).  I guess we could remove that, but I feel it blends in nicely in so much that if you're a person who believes that the certificates will come from another vector, you simply name the secret in the hosts array.  If you _also_ need to provide the data for that certificate, you simply provide it under `ingress.secrets` array.

Reply:
If this is confusing it might be worth having a section in the README that outlines how to leverage this for both cases.

Reply:
if @prydonius wants clarification, i'm willing to add it - LMK - and thanks!

Reply:
@venezia makes sense, but the indentation in values.yaml is incorrect, it's currently part of the hosts array (if I'm reading it correctly)